src/routing/queryParams.ts

```diff
@@ -1,3 +1,4 @@
+// TODO: rename QueryParams https://en.wikipedia.org/wiki/Query_string
```

Comment:
Query string is the whole part of the URL string containing Query Parameters. A single label value pair is called a query parameter, so I think the current naming is OK.